arm-web/2016-03-01/ResourceProvider.json

```diff
@@ -9,6 +9,70 @@
     "https"
   ],
   "paths": {
+    "/providers/Microsoft.Web/publishingUsers/web": {
+      "get": {
+        "summary": "Gets publishing user",
+        "description": "Gets publishing user",
+        "operationId": "GetPublishingUser",
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/User"
+            }
+          }
+        }
+      },
+      "put": {
+        "summary": "Updates publishing user",
+        "description": "Updates publishing user",
+        "operationId": "UpdatePublishingUser",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "requestMessage",
```

Comment:
If the parameter is named "userDetails" then this would be better. It gives better developer experience.